Governance/BRANDING.md

```diff
@@ -0,0 +1,46 @@
+## Branding Guidelines
+
+This file hosts the organization brand assets and guidelines for moja global. We expect contributors and maintainers to follow these guidelines during content promotion, outreach and communicating contribution efforts.
+
+## About
+
+Moja global is a not for profit, collaborative project that brings together a community of experts to develop open-source software that allows users to accurately and affordably estimate greenhouse gas emissions and removals from the AFOLU sector. The project’s members aim to support the widest possible use of credible emissions estimation software.
+
+## What we do?
+
+Moja global is building a community of experts, scientists and developers to create the first open-source source software for MRV systems for the land sector that is affordable, accurate and fully customizable to a user’s needs and available data, whether the user represents a country, a region, a local government, a company or a project.
+
+## Title
+
+**Moja global** is always written in all lower case, except when at the start of a sentence in which case it appears as **Moja global**. Moja global is written in lower case because **open source** is also the lower case (even though both are registered trademarks). While there is no published reason why moja global uses lower case, some have explained the use of lower case as a contrast to proper nouns and as an expression of the supportive role moja global tries to fulfil.
```

Comment:
```suggestion
**Moja global** is always written in all lower case, except when at the start of a sentence in which case it appears as **Moja global**. Moja global is written in lower case because **open source** is also the lower case (even though both are registered trademarks). While there is no published reason why moja global uses lower case, some have explained the use of lower case as a contrast to proper nouns and as an expression of the supportive role moja global tries to fulfill.
```